src/constants/shapes.js

```diff
@@ -53,3 +55,14 @@ export const topicItemShape = shape({
 export const topicShape = shape({
   topic: topicItemShape,
 });
+
+export const userShape = shape({
+  avatar: avatarShape,
+  email: string,
+  firstName: string,
+  fullName: string,
+  gender: string,
+  id: number,
+  lastName: string,
+  username: string,
```

Comment:
```suggestion
  username: string.isRequired,
```
remember you can do this if the value is required for the app to not crash